test/containerbuild.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
I think we just need `subprocess` and `pytest` here.

Reply:
Yeah, silly me - and a linter :)